indexing-service/src/main/java/org/apache/druid/indexing/common/task/AbstractBatchIndexTask.java

```diff
@@ -935,9 +935,34 @@ protected TaskReport.ReportMap buildLiveIngestionStatsReport(
     );
   }
 
+  protected TaskReport.ReportMap buildLiveIngestionStatsReportWithoutContext(
+      IngestionState ingestionState,
+      Map<String, Object> unparseableEvents,
+      Map<String, Object> rowStats
+  )
+  {
+    return TaskReport.buildTaskReports(
+        new IngestionStatsAndErrorsTaskReport(
+            getId(),
+            new IngestionStatsAndErrors(
+                ingestionState,
+                unparseableEvents,
+                rowStats,
+                null,
+                false,
+                0L,
+                null,
+                null,
+                null
+            )
+        )
```

Comment:
this part should be commoned out with the other method.

Reply:
Done